packages/form/src/CheckboxGroup.js

```diff
@@ -0,0 +1,62 @@
+import React, { createContext, useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useField, useFormikContext } from 'formik';
+import Feedback from './Feedback';
+import FormGroup from './FormGroup';
+
+export const CheckboxGroupContext = createContext();
+
+export const useCheckboxGroup = name => {
+  const { setFieldValue } = useFormikContext();
+  const { name: groupName, value = [], ...rest } = useContext(
+    CheckboxGroupContext
+  );
+
+  const toggle = () => {
+    const valueArray = [...value] || [];
```

Comment:
i think we can remove `|| []` from this line since even an empty array is truthy.